src/components/data-management/ProjectDetails.jsx

```diff
@@ -297,7 +298,6 @@ const ProjectDetails = ({ width, height }) => {
         value={text}
         onAfterSubmit={(name) => dispatch(updateSample(record.uuid, { name }))}
         onDelete={() => dispatch(deleteSamples(record.uuid))}
-        validationFunc={(name) => validateInputs(name, validationChecks, validationParams).isValid}
```

Comment:
Why isn't validation here required any more?

Reply:
This field will be replaced with a species dropdown. I deleted this line so I don't have to define `validationCheks`, which is replaced with `metadataNameValidation`